### PR TITLE
feat(scrape_pacer_free_opinions): apply task recap_document_into_opinions

### PR DESCRIPTION
**Pull request by grossir**
Features:
- add corpus_importer.tasks.recap_document_into_opinions into the scrape_pacer_free_opinions chain: this will make the recap documents be ingested in real time into the case law database
- tweaked the task so it can be used by both the scraper_pacer_free_opinions and recap_into_opinions command

Refactors:
- renamed task `ingest_recap_document` to `recap_document_into_opinions`
- renamed task `extract_recap_document` to `extract_recap_document_for_opinions`
- deleted duplicated function `extract_recap_document` in recap_into_opinions command file; delete unused imports and code

### CODE REVIEW

File: cl/corpus_importer/management/commands/recap_into_opinions.py [hunk 1]
```diff
@@ -89,6 +49,8 @@ def import_opinions_from_recap(
 
         # Manually select the replica db which has an addt'l index added to
         # improve this query.
```
Comment (mlissner) [inline cl/corpus_importer/management/commands/recap_into_opinions.py:51]: This is a one-time script, right? Let's make sure we remove that index when we're done. Can we make a sub-issue to remember to do that? We need to be really careful not to permanently add indexes we won't use in the future.

Comment (flooie) [reply cl/corpus_importer/management/commands/recap_into_opinions.py:51]: Well.. it was a one time script until it wasn't right?  Regardless yes we should.

Comment (mlissner) [reply cl/corpus_importer/management/commands/recap_into_opinions.py:51]: Great. Can we make that issue or sub-issue, please?

Comment (flooie) [reply cl/corpus_importer/management/commands/recap_into_opinions.py:51]: √

**Comment by flooie** [comment]
I'm not sure if this is ready or not - last I heard it was being checked still.  If thats not the case please let me know @grossir next time you are in.

**Comment by grossir** [comment]
I solved my Celery issues, and just finished testing the changes. Both `recap_into_opinions` and `scrape_pacer_free_documents` commands work with the introduced changes. 

We should introduce some tests, to help ensure that future changes do not change the desired behavior. Do you want me to do it on this PR, or should I create ? @flooie If not, this is ready for merging

--------------------

About Celery, it took me a while to understand what was failing. The `recap_into_opinions` command uses the "batch1" queue by default. When testing locally, one must [activate](https://github.com/freelawproject/courtlistener/wiki/Working-with-Celery-on-CourtListener) the celery workers manually. However, if one does not specify the queues they will target, they will only target the default queue named "celery". This caused the behavior I was observing, that tasks were sent, but not executed, since no worker was looking for them. The same was happening on `scrape_free_pacer_opinions`, which has a default queue name of "pacerdoc1". I have added comments to the wiki to warn a future developer/tester about this.

**Review by flooie — APPROVED** [review]
Looks good to me

**Comment by mlissner** [comment]
Heck yeah. Nice to have this going in.

**Comment by grossir** [comment]
This is working, we have [opinions from yesterday](https://www.courtlistener.com/?q=&type=o&order_by=dateFiled+desc&stat_Published=on&stat_Unknown=on&court=almd+alnd+alsd+akd+azd+ared+arwd+cacd+caed+cand+casd+cod+ctd+ded+flmd+flnd+flsd+gamd+gand+gasd+hid+idd+ilcd+ilnd+ilsd+innd+insd+iand+iasd+ksd+kyed+kywd+laed+lamd+lawd+med+mdd+mad+mied+miwd+mnd+msnd+mssd+moed+mowd+mtd+ned+nvd+nhd+njd+nmd+nyed+nynd+nysd+nywd+nced+ncmd+ncwd+ndd+ohnd+ohsd+oked+oknd+okwd+ord+paed+pamd+pawd+rid+scd+sdd+tned+tnmd+tnwd+txed+txnd+txsd+txwd+utd+vtd+vaed+vawd+waed+wawd+wvnd+wvsd+wied+wiwd+wyd+gud+nmid+prd+vid+californiad+illinoised+illinoisd+indianad+orld+ohiod+pennsylvaniad+southcarolinaed+southcarolinawd+tennessed+canalzoned&page=1), created around today's midnight. [Cluster object example](https://www.courtlistener.com/api/rest/v3/clusters/10266701/)

**Pull request merged**